var/spack/repos/builtin/packages/raja-perf/package.py

```diff
@@ -0,0 +1,248 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import socket
+import re
+
+from spack.package import *
+
+from .camp import blt_link_helpers, cuda_for_radiuss_projects, hip_for_radiuss_projects
+
+
+class RajaPerf(CachedCMakePackage, CudaPackage, ROCmPackage):
+    """RAJA Performance Suite."""
+
+    homepage = "https://github.com/LLNL/RAJAPerf"
+    git = "https://github.com/LLNL/RAJAPerf.git"
+
+    version("develop", branch="develop", submodules="True")
+    version("main", branch="main", submodules="True")
+    version("2023.06.0", tag="v2023.06.0", submodules="True")
+    version("2022.10.0", tag="v2022.10.0", submodules="True")
+    version("0.12.0", tag="v0.12.0", submodules="True")
+    version("0.11.0", tag="v0.11.0", submodules="True")
+    version("0.10.0", tag="v0.10.0", submodules="True")
+    version("0.9.0", tag="v0.9.0", submodules="True")
+    version("0.8.0", tag="v0.8.0", submodules="True")
+    version("0.7.0", tag="v0.7.0", submodules="True")
+    version("0.6.0", tag="v0.6.0", submodules="True")
+    version("0.5.2", tag="v0.5.2", submodules="True")
+    version("0.5.1", tag="v0.5.1", submodules="True")
+    version("0.5.0", tag="v0.5.0", submodules="True")
+    version("0.4.0", tag="v0.4.0", submodules="True")
```

Comment:
Should've mentioned this before but I didn't dig into the problem.  I get `All fetchers failed` for versions `0.4.0` and `0.5.1`.  

I briefly looked at the site last week and it seemed like they should work.

Reply:
It turns out 0.4.0 and 0.5.1 miss the "v".

I’m fixing this.

<img width="279" alt="Screen Shot 2023-08-28 at 7 05 34 PM" src="https://github.com/spack/spack/assets/51493078/9a80af0c-bd91-425a-b800-6ed20796f0d8">

Reply:
Fixed